src/app/main/component/events/components/create-edit-events/create-edit-events.component.scss

```diff
@@ -140,6 +140,8 @@
   padding: 40px 0 60px;
   display: flex;
   justify-content: flex-end;
+  align-items: start;
+  max-width: 1640.4px;
```

Comment:
1640.4 px don't match principle of Pixel Perfect Design, it is better to write value without decimal values

Reply:
Ok, I'll change to 1640